example/crons/event.processor.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { Cron } from '@nestjs/schedule';
+import { Locker } from '../utils/locker';
+import { EventProcessor } from "../../src/event.processor";
+
+@Injectable()
+export class EventProcessorService {
+  private readonly logger: Logger;
+  private lastTimestamp: number | undefined;
+
+  constructor() {
+    this.logger = new Logger(EventProcessorService.name);
+  }
+
+  @Cron('*/1 * * * * *')
+  async handleNewMultiversxEvents() {
+    Locker.lock('newMultiversxEvents', async () => {
+      const eventProcessor = new EventProcessor();
+      await eventProcessor.start({
+        elasticUrl: 'https://index.multiversx.com',
```

Comment:
Question: is possible to fetch some urls from a config file 🤔  ?

Reply:
no need for it. this is a simple nestjs example app